instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go

```diff
@@ -115,6 +179,15 @@ func (ct *clientTracer) start(hook, spanName string, attrs ...attribute.KeyValue
 }
 
 func (ct *clientTracer) end(hook string, err error, attrs ...attribute.KeyValue) {
+	if !ct.useSpans {
+		ct.root.AddEvent(hook + ".done")
+		if err != nil {
+			attrs = append(attrs, attribute.String(hook+".error", err.Error()))
+		}
+		ct.root.SetAttributes(attrs...)
```

Comment:
Should these attributes be added to the event rather than the span?

Reply:
whoops, fixed that in `start`, but forgot to also fix in `end`, I have updated the PR.  Thanks